etc/kayobe/kolla/globals.yml

```diff
@@ -52,7 +52,7 @@ keystone_tag: 9.0.1.2-1
 kibana_tag: 9.0.1.2-1
 kolla_toolbox_tag: 9.0.1.2-1
 logstash_tag: 9.0.1.2-1
-magnum_tag: 9.0.1.2-1
+magnum_tag: 9.1.0.1-1
```

Comment:
Did you mean to add kolla_openstack_release?

Reply:
Where did I add that?

Reply:
Yes I did, since I built a new container with the bumped up kolla_openstack_release

Reply:
A little funky that the tag is the same as the train containers, but I don't have a better suggestion. We normally bump the `-1` suffix if we make changes to kolla-build blocks that affect the images, but otherwise looks good to me.